apps/quality_assurance/schema.py

```diff
@@ -0,0 +1,35 @@
+import graphene
+
+from graphene_django import DjangoObjectType
+
+from quality_assurance.models import EntryReviewComment, EntryReviewCommentText
+from .enums import CommentTypeEnum
+
+
+class EntryReviewCommentTextType(DjangoObjectType):
+    class Meta:
+        model = EntryReviewCommentText
+        field = (
+            'text',
+            'created_at',
+            'id'
+        )
+
+
+class EntryReviewCommentType(DjangoObjectType):
+    class Meta:
+        model = EntryReviewComment
+        fields = (
+            'id',
+            'entry',
+            'created_by',
+            'created_at',
+            'mentioned_users'
+        )
+
+    comment_type = graphene.Field(CommentTypeEnum)
+    text_history = graphene.List(graphene.NonNull(EntryReviewCommentTextType))
```

Comment:
Add text as well (this will use the latest text). Fetch this from EntryReviewCommentLoader.

We will need to think about `text_history`, it's used rarely, so we should fetch it only when required. So the client will fetch it only when required, so we need to add a entryreviewcomment at project scope level as well (by id).